src/lib/vega-wallet/vega-wallet-service.ts

```diff
@@ -1,5 +1,3 @@
-import semver from "semver";
```

Comment:
I think this is the only place we use this package so it can be removed from package.json